lib/decidim/time_tracker/version.rb

```diff
@@ -3,7 +3,7 @@
 module Decidim
   # This holds the decidim-time_tracker version.
   module TimeTracker
-    VERSION = "0.3"
-    DECIDIM_VERSION = "0.23.1"
+    VERSION = "0.4"
+    DECIDIM_VERSION = "0.26.4"
```

Comment:
We prefer no to specify the decidim version and specify the compatible versions instead.
```suggestion
    DECIDIM_VERSION = "0.26.4"
    COMPAT_DECIDIM_VERSION = [">= 0.25.0", "< 0.27"].freeze
```
You will have to change the gem spec too using this `COMPAT_DECIDIM_VERSION` variable. For example: https://github.com/Platoniq/decidim-module-odoo/blob/release/0.26-stable/decidim-odoo.gemspec

Reply:
Hi @marinamuca, thanks for taking the time to review my PR. 
Will check the items you pointed out this week.